flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -671,8 +671,11 @@ static void go_enable_fly_home()
 		vtol_fsm_path_desired.End[2] = -RTH_MIN_ALTITUDE;
 	}
 
-	vtol_fsm_path_desired.StartingVelocity = RTH_VELOCITY;
-	vtol_fsm_path_desired.EndingVelocity = RTH_VELOCITY;
+	float rth_vel = 3.0f;
```

Comment:
IIRC , historically we've not assigned a default value when the UAVO variable is declared, in order to save flash. Since the variable is immediately overwritten by ```VtolPathFollowerSettingsReturnToHomeVelGet()```, it would seem safe to do the same here.